package/MDAnalysis/analysis/contacts.py

```diff
@@ -153,252 +153,11 @@
 import MDAnalysis
 import MDAnalysis.lib.distances
 from MDAnalysis.lib.util import openany
-
+from MDAnalysis.analysis.distances import distance_array
+from .base import AnalysisBase
 
 logger = logging.getLogger("MDAnalysis.analysis.contacts")
 
-
-class ContactAnalysis(object):
```

Comment:
I put `ContactAnalysis` back in jandom/mdanalysis#1 – it should not have been removed. Raise an issue to overhaul `ContactAnalysis` to work within the new analysis model. We can then also rename it and break it's API, or for all I care, completely rewrite with more focused functionality. But for right now it has to stay.